CHANGELOG.md

```
@@ -49,6 +49,12 @@ _July 03, 2026_
 - `message.polls.accessibility.*` — VoiceOver labels for polls
 - `channel.item.accessibility.*` — VoiceOver labels for channel list items
 
+# [4.101.2](https://github.com/GetStream/stream-chat-swiftui/releases/tag/4.101.2)
+_July 03, 2026_
+
+### 🐞 Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)